test/unit/client/adminusers_client/find_service_test.js

```diff
@@ -0,0 +1,108 @@
+'use strict'
+
+// NPM dependencies
+const path = require('path')
+const Pact = require('pact')
+const { expect } = require('chai')
+const proxyquire = require('proxyquire')
+
+// Custom dependencies
+const PactInteractionBuilder = require('../../../fixtures/pact_interaction_builder').PactInteractionBuilder
+const serviceFixtures = require('../../../fixtures/service_fixtures')
+
+const ADMINUSERS_RESOURCE = '/v1/api/services'
+const port = Math.floor(Math.random() * 48127) + 1024
+
+function getAdminusersClient (baseUrl = `http://localhost:${port}`) {
+  return proxyquire('../../../../app/services/clients/adminusers_client', {
+    '../../../config': {
+      ADMINUSERS_URL: baseUrl
+    }
+  })
+}
+
+describe('adminusers client - find a service associated with a particular gateway account id', function () {
+  const provider = Pact({
+    consumer: 'products-ui-to-be',
+    provider: 'adminusers',
+    port: port,
+    log: path.resolve(process.cwd(), 'logs', 'mockserver-integration.log'),
+    dir: path.resolve(process.cwd(), 'pacts'),
+    spec: 2,
+    pactfileWriteMode: 'merge'
+  })
+
+  let gatewayAccountId, response, result
+
+  before(() => provider.setup())
+  after((done) => provider.finalize().then(done()))
+
+  describe('when a service is found', () => {
+    const customBranding = {
+      css_url: 'https://example.org/mycss',
+      image_url: 'https://example.org/myimage'
+    }
+    before(done => {
+      const adminusersClient = getAdminusersClient()
+      gatewayAccountId = 123456
+      const serviceData = {
+        gateway_account_ids: [gatewayAccountId],
+        custom_branding: customBranding
+      }
+
+      response = serviceFixtures.validServiceResponse(serviceData)
+
+      const interaction = new PactInteractionBuilder(`${ADMINUSERS_RESOURCE}`)
+        .withQuery('gatewayAccountId', `${gatewayAccountId}`)
```

Comment:
same here

Reply:
👍

Reply:
As clarified with @SandorArpa  ``${gatewayAccountId}`` converts `gatewayAccountId` to a string and we need string there